client/me/purchases/manage-purchase/index.tsx

```diff
@@ -1056,9 +1081,40 @@ class ManagePurchase extends Component<
 		const registrationAgreementUrl = getDomainRegistrationAgreementUrl( purchase );
 		const domainRegistrationAgreementLinkText = translate( 'Domain Registration Agreement' );
 
+		const helpOptions = {
+			components: {
+				strong: <strong />,
+				a: <a href={ localizeUrl( SUPPORT_ROOT ) } rel="noopener noreferrer" target="_blank" />,
+			},
+		};
+
+		const cancelOptions = {
+			components: {
+				strong: <strong />,
+				a: <a href={ localizeUrl( DOMAIN_CANCEL ) } rel="noopener noreferrer" target="_blank" />,
+			},
+		};
+		const supportText = translate(
+			' Need help? {{a}}Get in touch with one of our Happiness Engineers{{/a}}.',
+			helpOptions
+		);
+
+		const cancelText = translate(
+			' Cancel Domain and Refund? Please {{a}}click here.{{/a}}',
```

Comment:
It's not a good idea to start translated strings with a space. You can instead use CSS or just literal spaces between strings for layout purposes.

Reply:
Interesting! 🤔 I've never done it that way, but... if that works, I'm fine with it. Typically what I would do is put a space into the JSX. A trivial example would be `<element1/> <element2/>` but since you need both elements to be conditional it would probably have to become `{ purchase.productType === 'domain_transfer' && cancelText }{ ' ' }{ purchase.productType === 'domain_transfer' && domainTransferDuration }`, which is kind of awkward. 

On the other hand, since they're the same condition you could do this instead:

```js
{ purchase.productType === 'domain_transfer' && (
	<>
		{ cancelText } { domainTransferDuration }
	</>
) }
```

Anyway, I'll leave it up to you.